### PR TITLE
Added management command to populate history as of 06/29.

### PR DESCRIPTION
**Pull request by HassanJaveed84**

### CODE REVIEW

File: ecommerce/core/management/commands/backfill_history.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                        count = cursor.fetchone()[0]
+
+                    if count == len(rows):
+                        log.info(
+                            u"Initial history records already exist for ids %s..%s - skipping.",
+                            ','.join(row_ids[:2]), ','.join(row_ids[-2:])
+                        )
+                        continue
+                    elif count != 0:
+                        raise Exception(u"Database count: %s does not match input count: %s" % (count, len(ids)))
```
Comment (pwnage101) [inline ecommerce/core/management/commands/backfill_history.py]: ids is just a string, in think.  You probably want row_ids or rows:

```suggestion
                        raise Exception(u"Database count: %s does not match input count: %s" % (count, len(rows)))
```

File: ecommerce/core/management/commands/backfill_history.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                    values = self.replace_values(values, 'NULL', None)
+                    # Replace 'true' with True
+                    values = self.replace_values(values, 'true', True)
+                    # Replace 'false' with False
+                    values = self.replace_values(values, 'false', False)
+                    # Add history columns data
+                    for value in values:
+                        value.extend([history_date, self.HISTORY_CHANGE_REASON, '+', self.HISTORY_USER_ID])
+                    # Convert to tuple
+                    values = [tuple(value) for value in values]
```
Comment (pwnage101) [inline ecommerce/core/management/commands/backfill_history.py]: Nit:
```suggestion
                    values = [tuple(row) for row in values]
```

**Review by pwnage101 — CHANGES_REQUESTED** [review]

**Review by pwnage101 — APPROVED** [review]

**Review by bmedx — APPROVED** [review]
Seems ok to me, what are the commented out tables?

**Comment by HassanJaveed84** [comment]
I commented out the ones with no missing historical records. I'll remove them.

**Comment by HassanJaveed84** [comment]
@cpennington please review and merge.

**Comment by cpennington** [comment]
I'm having some deja vu. Didn't I already review this command?

**Review by cpennington — APPROVED** [review]

**Comment by HassanJaveed84** [comment]
That was the Step 1 out of 2 of history population. This is the second step.
I had created a PR to remove it, please merge(I just rebased it): 
https://github.com/edx/ecommerce/pull/2615

**Pull request merged**